schema/products/ibu10m.json

```diff
@@ -0,0 +1,309 @@
+        },
+        "stac_extensions": {
+          "const": [
+            "https://stac-extensions.github.io/projection/v1.1.0/schema.json"
+          ]
+        },
+        "type": {
+          "const": "Feature"
+        },
+        "id": {
+          "type": "string"
+        },
+        "bbox": {
+          "type": "array",
+          "minItems": 4,
+          "maxItems": 4
+        },
+        "properties": {
+          "type": "object",
+          "required": [
+            "description",
+            "datetime",
+            "start_datetime",
+            "end_datetime",
+            "created",
+            "providers",
+            "proj:epsg",
+            "proj:bbox",
+            "proj:shape"
+          ],
+          "properties": {
+            "providers": {
+              "const": [
+                {
+                  "name": "Copernicus Land Monitoring Service",
+                  "description": "The Copernicus Land Monitoring Service provides geographical information on land cover and its changes, land use, ground motions, vegetation state, water cycle and Earth's surface energy variables to a broad range of users in Europe and across the World in the field of environmental terrestrial applications.",
+                  "roles": ["licensor", "host"],
+                  "url": "https://land.copernicus.eu"
+                }
+              ]
+            }
+          }
+        },
+        "collection": {
+          "const": "imperviousness-built-up-10m"
+        },
+        "links": {
+          "type": "array",
+          "allOf": [
+            {
+              "contains": {
+                "const": {
+                  "rel": "license",
+                  "href": "https://land.copernicus.eu/en/data-policy"
+                }
+              }
+            },
+            {
+              "contains": {
+                "type": "object",
+                "properties": {
+                  "rel": {
+                    "const": "self"
+                  }
+                }
+              }
+            },
+            {
+              "contains": {
+                "type": "object",
+                "properties": {
+                  "rel": {
+                    "const": "root"
+                  }
+                }
+              }
+            },
+            {
+              "contains": {
+                "type": "object",
+                "properties": {
+                  "rel": {
+                    "const": "parent"
+                  }
+                }
+              }
+            },
+            {
+              "contains": {
+                "type": "object",
+                "properties": {
+                  "rel": {
+                    "const": "collection"
+                  }
+                }
+              }
+            }
+          ]
+        },
+        "assets": {
+          "type": "object",
+          "minProperties": 3,
+          "maxProperties": 3,
+          "propertyNames": {
```

Comment:
Since all items have the same assets, which are builtup_map, builtup_map_database, and builtup_map_worldfile, I would use `required` here.